virt/install/preparing-cluster-for-virt.adoc

```diff
@@ -41,6 +41,12 @@ Without an external monitoring system or a qualified human monitoring node healt
 
 * If your cluster uses worker nodes from multiple CPU vendors, live migration failures can occur. For example, a virtual machine with an AMD CPU might attempt to live-migrate to a node with an Intel CPU and likely fail migration. To avoid this, label nodes with a vendor-specific label, such as `Vendor=Intel` or `Vendor=AMD`, and set node affinity on your virtual machines to ensure successful migration. See xref:../../nodes/scheduling/nodes-scheduler-node-affinity.adoc#nodes-scheduler-node-affinity-configuring-required_nodes-scheduler-node-affinity[Configuring a required node affinity rule] for more information.
 
+* All CPUs must have Red Hat Enterprise Linux 8 and support the following requirements:
```

Comment:
Suggest rewriting this for clarity - "All CPUs must be supported by RHEL 8 and meet the following requirements:"

Reply:
Earlier, I had text which mentioned what you suggested "All CPUs must be supported by RHEL 8..." and Fabian noted the following in this PR earlier:

Fabian's note:
This is technically a little inprecise, because not the CPU is supporting RHEL 8, but RHEL 8 is supporting the CPU.
But it might be good enough

@lmandavi lmandavi 21 hours ago Author Contributor
Let me redo the text for clarity, thanks for the clarification!

@sjhala-ccs - so let me know your thoughts. Can we leave the sentence as it is currently stated. If not, how do you recommend we reposition it?

Reply:
@lmandavi As far as I understand my suggested rewrite aligns with what Fabian said earlier that the CPU is not supporting RHEL 8, but RHEL 8 is supporting the CPU.

The sentence "All CPUs must be supported by RHEL 8 and meet the following requirements" conveys that aspect more clearly than the current version IMO.